config.example.localhost.yaml

```diff
@@ -0,0 +1,116 @@
+# RESTBase config for small wiki installs
+#
+# - sqlite backend
+# - parsoid at http://localhost:8142
+# - wiki at http://localhost/w/api.php
+# 
+# Quick setup:
+# - npm install
+# # needs libsqlite3-dev:
+# - npm install restbase-mod-table-sqlite
```

Comment:
Strangely, it did not for me the first time round. It did work in the second attempt, though.